tensorflow/compiler/xla/service/gpu/ir_emitter.cc

```diff
@@ -164,8 +182,19 @@ Status IrEmitter::EmitCallToNestedComputation(
     emitted_function = ir_emitter_nested.GetEmittedFunction();
   }
 
-  std::vector<llvm::Value*> arguments(operands.begin(), operands.end());
-  arguments.push_back(output);
+  // For AMDGPU target, may need to addrspacecast alloca variables from
+  // addrspace 5 to addrspace 0
+  std::vector<llvm::Value*> arguments;
+  for (auto& arg : operands) {
```

Comment:
This can be written somewhat more explicitly using absl::c_transform, e.g. https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/client/xla_builder.cc#L721

Reply:
Done, thanks for the feedback